src/components/tabs/__internal__/tabs-header/tabs-header.component.js

```diff
@@ -41,6 +46,10 @@ TabsHeader.propTypes = {
   alternateStyling: PropTypes.bool,
   noRightBorder: PropTypes.bool,
   isInSidebar: PropTypes.bool,
+  /**
+   * @ignore @private
```

Comment:
This doesn't need to have `@ignore @private` on it because the whole component is internal